### PR TITLE
Improvement/documentation-refactoring

### PR DESCRIPTION
**Pull request by AdrienRem**

### CODE REVIEW

**Comment by AdrienRem** [comment]
rhaaaaaa CC count the javadocs (package-info) as code so it decreases the code coverage

File: app/src/main/java/ch/epfl/sdp/contamination/fragment/InfectionFragment.java [hunk 1]
```diff
@@ -28,6 +28,9 @@
 
 import static android.content.Context.BIND_AUTO_CREATE;
 
+/**
+ * Say what this class is doing
```
Comment (Ulydev) [inline app/src/main/java/ch/epfl/sdp/contamination/fragment/InfectionFragment.java]: The class is pretty self-explanatory but we should still have an introduction line.
* View displaying the probability of infection of the user

**Review by Ulydev — CHANGES_REQUESTED** [review]
Great PR, makes the codebase more understandable for a newcomer. Good job detailing each package-info. I submitted one tiny change for InfectionFragment to ensure meaningful comments.

**Review by TheCrazyKing — APPROVED** [review]
I read through your changes and everything seems legit. nice work.

File: app/src/main/java/ch/epfl/sdp/utilities/package-info.java [hunk 2]
```diff
@@ -0,0 +1,4 @@
+/**
+ * Will deleted soon and the contents of Tools will move in other folders
```
Comment (TheBlueHawk) [inline app/src/main/java/ch/epfl/sdp/utilities/package-info.java:2]: Where would you like to move them?

**Review by TheBlueHawk — COMMENTED** [review]
Good refactoring, but what about the utilities classes?

File: app/src/main/java/ch/epfl/sdp/utilities/package-info.java (hunk 2 above)
Comment (AdrienRem) [reply app/src/main/java/ch/epfl/sdp/utilities/package-info.java:2]: I'd put the biometric stuff inside or in the package where we deal with biometrics and the others functions can go in the Coronagame class

File: app/src/main/java/ch/epfl/sdp/contamination/fragment/InfectionFragment.java (hunk 1 above)
Comment (AdrienRem) [reply app/src/main/java/ch/epfl/sdp/contamination/fragment/InfectionFragment.java]: done !

File: app/src/main/java/ch/epfl/sdp/utilities/package-info.java (hunk 2 above)
Comment (AdrienRem) [reply app/src/main/java/ch/epfl/sdp/utilities/package-info.java:2]: The SingleFragmentActivity can also go in the testing folder after some small changes in the main folder but Matteo's merge is already going to be very difficult. I don't want to make it worse

**Review by Ulydev — APPROVED** [review]

**Pull request merged**